docs/page-controller.md

```diff
@@ -88,13 +87,16 @@ Registering a React-powered page is similar to connecting a PHP page, but with s
 
 Register pages with `wc_admin_register_page()` using these parameters:
 
-* `id` - Identifies the page with the controller. Required.
-* `parent` - Denotes the page as a child of `parent`. Used for breadcrumbs. Optional.
-* `title` - Page title. Used to build breadcrumbs. String or array of breadcrumb pieces. Required.
-* `path` - Page path (relative to `#wc-admin`). Used for identifying this page and for linking breadcrumb pieces when this page is a `parent`. Required.
-* `capability` - User capability needed to access this page. Optional (defaults to `manage_options`).
-* `icon` - Dashicons helper class or base64-encoded SVG. Optional.
-* `position` - Menu item position for parent pages. Optional. See: `add_menu_page()`.
+-   `id` - Identifies the page with the controller. Required.
+-   `parent` - Denotes the page as a child of `parent`. Used for breadcrumbs. Optional.
+-   `title` - Page title. Used to build breadcrumbs. String or array of breadcrumb pieces. Required.
+-   `path` - Page path (relative to `#wc-admin`). Used for identifying this page and for linking breadcrumb pieces when this page is a `parent`. Required.
+-   `capability` - User capability needed to access this page. Optional (defaults to `manage_options`).
+-   `icon` - Dashicons helper class or base64-encoded SVG. Optional.
```

Comment:
Should we note that this won't get displayed in the new Navigation? Also, I got stuck for a moment thinking I needed to omit the `dashicons-` prefix for the name. Perhaps that's also worth a mention (to use the full class name)?

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/5844/commits/cd16085eb9ade81d74fd8894266c2cd6038738c1